source/Mortgage.cpp

```diff
@@ -27,17 +27,17 @@ using namespace std;
 
 // Find out how much you can afford to borrow with the given annual revenue
 // and the given credit score (which should be between 200 and 800).
-int64_t Mortgage::Maximum(int64_t annualRevenue, int creditScore, int64_t currentPayments)
+int64_t Mortgage::Maximum(int64_t annualRevenue, int creditScore, double currentPayments)
 {
-	const int term = 365;
-	annualRevenue -= term * currentPayments;
-	if(annualRevenue <= 0)
+	const static int term = 365;
```

Comment:
```suggestion
	static const int term = 365;
```
Though, marking this as `static` won't make a difference after compiler optimisations.

Reply:
True enough.  I don't normally mark method invariants as static unless I need the value to stick around between invocations, but I think someone else suggested it.

Reply:
Yea me, but you are right warp.

Anyways now that I think about it, this should be moved to the unnamed namespace anyways as it is a constant that is not getting calculated on the flight. That's what we normally do, right?

Reply:
Yup

Reply:
Changed.